package.json

```diff
@@ -33,16 +33,19 @@
     "check": "prettier --check . && tsc --noEmit --allowJs --lib es2017,dom ./src/webxdc.js"
   },
   "dependencies": {
-    "@vitejs/plugin-basic-ssl": "^1.1.0",
-    "@vitejs/plugin-legacy": "^6.0.0",
+    "@vitejs/plugin-basic-ssl": "^1.2.0",
+    "@vitejs/plugin-legacy": "^6.0.1",
     "eruda": "^3.4.1",
     "terser": "^5.36.0",
     "vite-plugin-zip-pack": "^1.2.4"
   },
+  "peerDependencies": {
+    "vite": "^6.0.0"
+  },
   "devDependencies": {
     "@webxdc/types": "latest",
     "prettier": "latest",
     "typescript": "latest",
-    "vite": "latest"
+    "vite": "^6.1.0"
```

Comment:
generally better practice to specify a more explicit version range. can revert if not desired though